syntax = "proto3";

package orders;

import "google/protobuf/wrappers.proto"

service OrderService {
  rpc CreateOrder(CreateOrderRequest) returns (OrderResponse) {}
  rpc GetOrderResult(GetOrderRequest) returns (OrderResponse) {}
}

message CreateOrderRequest {
  int32 order_id = 1;
}

message GetOrderRequest {
  int32 order_id = 1;
  string skb_id = 2;
}

message OrderResponse {
  int32 order_id = 1;
  string skb_id = 2;
  string status = 3;
  string error = 4;
}